tests/tcp_proxy.py

```
@@ -0,0 +1,224 @@
+                                        name="proxy-%s:%d" % (self.listen_host, self.listen_port))
+        self._thread.start()
+        log.info("TcpProxy started %s:%d -> %s:%d",
+                 self.listen_host, self.listen_port,
+                 self.target_host, self.target_port)
+
+    def stop(self):
+        if self._server_sock:
+            try:
+                self._server_sock.close()
+            except Exception:
+                pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)